src/sandstorm/app-index/indexer.c++

```diff
@@ -389,7 +394,7 @@ void Indexer::updateIndexInternal(kj::StringPtr outputFilename, kj::StringPtr ou
     kj::FdOutputStream(file.getFd()).write(text.begin(), text.size());
     file.finalize(kj::str(outputDir, "/", appEntry.first, ".json"));
 
-    if (approvedApps) {
+    if (!experimental) {
```

Comment:
This symlink is written to keep track of the latest published package version so that we can reject duplicates. It doesn't apply to experimental because if we reject an experimental app for trivial reasons and it is resubmitted, it is allowed to have the same version number.